angular/projects/spark-angular/src/lib/components/sprk-flag/sprk-flag.component.ts

```diff
@@ -32,14 +28,32 @@ export class SprkFlagComponent {
    * container of the media-slot.
    */
   @Input()
-  additionalMediaClasses: string;
+  mediaAdditionalClasses: string;
 
   /**
    * Expects a space separated string
    * of classes to be added to the
    * container of the body-slot.
    */
   @Input()
+  bodyAdditionalClasses: string;
+
+  /**
+   * Deprecated: Expects a space separated string
+   * of classes to be added to the
+   * container of the media-slot.
+   * To be removed next release.
+   */
+  @Input()
+  additionalMediaClasses: string;
+
+  /**
+   * Deprecated: Expects a space separated string
```

Comment:
In the deprecated message, lets add what they should be using instead.

Reply:
fixed! 
![image](https://user-images.githubusercontent.com/4342363/89340858-622bb680-d66e-11ea-9dcf-7f66116734cf.png)